src/Nowruz/pages/setting/sections/account/account.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button } from 'src/components/atoms/button/button';
+import { Input } from 'src/Nowruz/modules/general/components/input/input';
+import { Modal } from 'src/components/templates/modal/modal';
+import { FeaturedIcon } from 'src/Nowruz/modules/general/components/featuredIcon-new';
+
+import { useRef, useState,FormEvent } from 'react';
+import css from './account.module.scss';
+import {
+    User,Organization,identities
+  } from 'src/core/api';
+import { SearchDropdown } from 'src/Nowruz/modules/general/components/SearchDropdown';
+
+import { useSelector } from 'react-redux';
+import store, { RootState } from 'src/store';
+import { Textarea } from 'src/components/atoms/textarea/textarea';
+import { deleteAccount } from 'src/pages/delete-profile/delete-profile.service';
+
+
+
+const Account = () => {
+
+    const identities = useSelector<RootState, CurrentIdentity[]>((state) => {
+        console.log('state',state.identity.entities)
+        return state.identity.entities;
+    });
+    const primary = identities.find((i) => i.primary);
+    console.log(primary);
+    
+
+    const user = useSelector<RootState, User | Organization | undefined>((state) => {
+        console.log('user',state.identity.entities.find((i)=> i.current === true))
+        return state.identity.entities.find((i)=> i.current === true)
+    }) as User;
+
+    const [modalVisibility, setModalVisibility] = useState(false);
+    let reasonbody = "";
+    const onChangeTextHandler = (e:any) =>{
+        console.log('value', e.target.value);
```

Comment:
Same here ;)

Reply:
@marjan-ara  i remove components which are out of Nowruz and i used Nowruz components.